src/vos/tests/SConscript

```diff
@@ -14,7 +14,7 @@ def scons():
     denv.AppendUnique(RPATH=[Literal(r'\$$ORIGIN/../lib/daos_srv')])
 
     vos_test_src = ['vos_tests.c', 'vts_io.c', 'vts_pool.c', 'vts_container.c',
-                    denv.Object("vts_common.c"), 'vts_aggregate.c',
+                    denv.Object("vts_common.c"), 'vts_aggregate.c', 'vts_dtx.c',
```

Comment:
Note to self, why does vts_common.c get built as an object here ?

Reply:
Honestly, I do not know the answer. It is introduced via the patch 6c34f47d937c4f34403e6aa3715647161962a42e